package.json

```diff
@@ -27,6 +27,7 @@
     "fs": "^0.0.1-security",
     "gray-matter": "^4.0.3",
     "less": "^4.1.1",
+    "lodash.throttle": "^4.1.1",
```

Comment:
I think you could add `lodash-es` module instead. That way we could keep using other features.